### ISSUE TITLE
Run-AlPipeline.ps1 generates a file BuildOutput.txt when compiling apps

### ISSUE THREAD

**Issue by eclipses** [microsoft/navcontainerhelper#2442]
HI Freddy, 

This must be because of some recent change.

When I run the command Run-AlPipeline.ps1 it generates a file BuildOutput.txt

According with the documentation of the function by default it should not generate a file. 

```
.Parameter buildOutputFile
Filename in which you want the build output to be written. Default is none, meaning that build output will not be written to a file, but only on screen.
```

**Comment by freddydk** [comment]
Sounds like a bug

**Comment by freddydk** [comment]
Shipped in 3.0.8
